mingw-w64-xpdf/PKGBUILD

```diff
@@ -5,7 +5,7 @@ pkgbase="mingw-w64-${_realname}-tools"
 pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}-tools")
 #conflicts=("${MINGW_PACKAGE_PREFIX}-poppler")
 pkgver=4.00
-pkgrel=1
```

Comment:
I'm not sure that we even need to bump the pkgrel here.

Reply:
Since the package itself stays the same, there's probably no need to bump indeed. I'll take it out.